db.php

```diff
@@ -435,6 +435,12 @@ public function db_connect( $query = '' ) {
 			if ( ! $this->ex_mysql_select_db( DB_NAME, $this->dbh ) ) {
 				return $this->log_and_bail( 'We were unable to select the database' );
 			}
+
+			// Set the SQL mode 
+			if ( is_callable( $this, 'set_sql_mode' ) ) {
+				$this->set_sql_mode();
+			}
+
```

Comment:
This code block had no effect on my test setup, neither with the fix to the `is_callable` function arguments, nor as is.